packages/stashjs/CHANGELOG.md

```diff
@@ -7,11 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## [0.6.1]
+
 ## Fixed
 
 - Filter out incompatible collections rather than returning an error when listing collections.
   This is a short-term fix for incompatibilities in ciphertext formats between StashJS and StashRB.
 
+## [0.6.0]
```

Comment:
Noticed that we're missing an entry here. Anyone know what was included in this release? @bennetthardwick or @ragibkl maybe?

Reply:
I'm not sure 🤷🏻

Reply:
yeah, I think I did a release, but did not update changelog. My bad.
It was mostly to make sure we can publish from new repo.